src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.tsx

```diff
@@ -4,15 +4,21 @@ import styled from '@emotion/styled';
 import isPropValid from '@emotion/is-prop-valid';
 
 import Radio from 'app/components/radio';
+import space from 'app/styles/space';
 
 type RadioGroupProps<C extends string> = {
-  value: string | number | null;
-
-  // An array of [id, name, description]
-  choices: [C, React.ReactNode, React.ReactNode?][];
-  disabled?: boolean;
   label: string;
+  disabled?: boolean;
+  /**
+   * An array of [id, name, description]
+   */
+  choices: [C, React.ReactNode, React.ReactNode?][];
+  value: string | number | null;
   onChange: (id: C, e: React.FormEvent) => void;
+  /**
+   * Switch the radio items to flow left to right, instead of vertically.
+   */
+  orientInline?: boolean;
```

Comment:
What kind of ordering are you trying to do here?

Reply:
That's a good question that I don't have a good answer for.

I think they got moved around just a bit from me trying to figure out what made sense. I think the only logic I have here is that `choices` `value` and `onChange` should be near each other